### PR TITLE
Remove or implement a large number of unused procs and vars

### PR DESCRIPTION
**Pull request by out-of-phaze**
## Description of changes
Removes a large number of unused procs and vars, and implements a few.

## Why and what will this PR improve
Overall code cleanup and quality pass.

### CODE REVIEW

File: nebula.dme [hunk 1]
```diff
@@ -1695,7 +1690,6 @@
 #include "code\modules\aspects\aspects_prosthetic_limbs.dm"
 #include "code\modules\aspects\aspects_prosthetic_organs.dm"
 #include "code\modules\assembly\assembly.dm"
-#include "code\modules\assembly\helpers.dm"
```
Comment (comma) [inline nebula.dme:1705]: This intended? File itself still exists

Comment (out-of-phaze) [reply nebula.dme:1705]: meant to be deleted

File: code/modules/admin/secrets/fun_secrets/break_all_lights.dm [hunk 2]
```diff
@@ -4,4 +4,5 @@
 /datum/admin_secret_item/fun_secret/break_all_lights/execute(var/mob/user)
 	. = ..()
 	if(.)
-		lightsout(0,0)
+		for(var/obj/machinery/power/apc/apc in SSmachines.machinery)
+			apc.overload_lighting()
```
Comment (MistakeNot4892) [inline code/modules/admin/secrets/fun_secrets/break_all_lights.dm:8]: Might be worth adding a tick check here while you're in the region.

File: code/modules/admin/secrets/fun_secrets/break_some_lights.dm [hunk 3]
```diff
@@ -1,7 +1,27 @@
...
+			epicentreList += pick(possibleEpicentres)
+		else
+			break
+
+	if(!length(epicentreList))
+		return
+
+	for(var/obj/abstract/landmark/epicentre in epicentreList)
+		for(var/obj/machinery/power/apc/apc in range(epicentre,lightsoutRange))
+			apc.overload_lighting()
```
Comment (MistakeNot4892) [inline code/modules/admin/secrets/fun_secrets/break_some_lights.dm:27]: As above, tick check here might be good.

File: code/modules/events/event_dynamic.dm [hunk 4]
```diff
@@ -1,88 +1,3 @@
...
-	possibleEvents[/datum/event/location_event/mundane_news] = 300
-	possibleEvents[/datum/event/trivial_news] = 400
-	possibleEvents[/datum/event/location_event] = 300
-
-	possibleEvents[/datum/event/money_lotto] = max(min(5, global.player_list.len), 50)
-	if(account_hack_attempted)
-		possibleEvents[/datum/event/money_hacker] = max(min(25, global.player_list.len) * 4, 200)
-
-
-	possibleEvents[/datum/event/carp_migration] = 20 + 10 * active_with_role[ASSIGNMENT_ENGINEER]
```
Comment (MistakeNot4892) [inline code/modules/events/event_dynamic.dm:30]: When the economy stuff and cargo stuff is in a better place I'd like to reimplement a system like this one to shift economic values based on news events. Future TODO though, this has been sitting there disconnected for years.

File: code/modules/mob/living/carbon/human/human.dm [hunk 5]
```diff
@@ -1,7 +1,6 @@
 /mob/living/carbon/human
 	name = "unknown"
 	real_name = "unknown"
-	voice_name = "unknown"
```
Comment (MistakeNot4892) [inline code/modules/mob/living/carbon/human/human.dm:4]: 95% sure this is meant to be returned to `GetVoice()` as a default.

File: code/modules/mob/living/carbon/human/human_attackhand.dm [hunk 6]
```diff
@@ -275,9 +275,6 @@
 	// Again.
 	start_compressions(H, FALSE, cpr_mode)
 
-/mob/living/carbon/human/proc/afterattack(atom/target, mob/living/user, inrange, params)
```
Comment (MistakeNot4892) [inline code/modules/mob/living/carbon/human/human_attackhand.dm:278]: Lol what on Earth?

File: code/modules/reagents/chems/chems_medicines.dm [hunk 7]
```diff
@@ -156,16 +156,18 @@
 	uid = "chem_immunobooster"
 
 /decl/material/liquid/immunobooster/affect_blood(var/mob/living/M, var/removed, var/datum/reagents/holder)
-	if(ishuman(M) && REAGENT_VOLUME(holder, type) < REAGENTS_OVERDOSE)
-		var/mob/living/carbon/human/H = M
-		H.immunity = min(H.immunity_norm * 0.5, removed + H.immunity) // Rapidly brings someone up to half immunity.
+	if(REAGENT_VOLUME(holder, type) >= REAGENTS_OVERDOSE)
```
Comment (MistakeNot4892) [inline code/modules/reagents/chems/chems_medicines.dm:159]: Was this immunity stuff meant to be a separate PR? I don't mind much but it seems a bit out of place.

File: code/modules/mob/living/carbon/human/human_attackhand.dm (hunk 6 above)
Comment (MistakeNot4892) [reply code/modules/mob/living/carbon/human/human_attackhand.dm:278]: Blame almost gave me a heart attack but it's actually over a decade old. Very odd.

File: code/modules/mob/living/carbon/human/human.dm (hunk 5 above)
Comment (MistakeNot4892) [reply code/modules/mob/living/carbon/human/human.dm:4]: Not blocking, just commentary.

**Review by MistakeNot4892 — DISMISSED** [review]
Left some small comments but generally this looks good to me.

File: code/modules/reagents/chems/chems_medicines.dm (hunk 7 above)
Comment (out-of-phaze) [reply code/modules/reagents/chems/chems_medicines.dm:159]: `adjust_immunity()` was unused

**Comment by out-of-phaze** [comment]
you reviewed before i was done, i forgor to push

File: code/modules/events/wormholes.dm [hunk 8]
```diff
@@ -35,7 +35,7 @@
 		wormholes += create_wormhole(enter, exit)
 
 /datum/event/wormholes/announce()
-	command_announcement.Announce("Space-time anomalies detected on the station. There is no additional data.", "[location_name()] Sensor Array", zlevels = affecting_z)
+	global.using_map.space_time_anomaly_detected_annoncement(affecting_z)
```
Comment (MistakeNot4892) [inline code/modules/events/wormholes.dm:38]: `annoncement`...

**Review by MistakeNot4892 — DISMISSED** [review]

**Review by MistakeNot4892 — APPROVED** [review]
